src/components/ArticleCard.jsx

```diff
@@ -1,6 +1,38 @@
 import React, { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 
+
+
+function Comments({ articleId }) {
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
```

Comment:
i would definitely investigate using axios. It's great to be able to use fetch as it will always work, but once using a library like React which will bundle for us, axios is definitely worth using